Gemfile

```diff
@@ -2,15 +2,19 @@ source 'https://rubygems.org'
 
 gemspec
 gem 'fog-aws'
-gem 'capybara', '~> 2.3.0'
-gem 'database_cleaner', '~> 0.9.1'
-gem 'rspec-rails', '~> 3.2.0'
-gem 'poltergeist'
 gem 'ammeter'
-gem 'test-unit', platforms: [:ruby_22]
 
-group :development do
+group :development, :test do
```

Comment:
That change must have been lost in a rebase. I think we should revisit `d22e903db0ea96f63df80fd58038509033c0fe49` as I don't see the reason for including test gems in all environments.